examples/trim.rs

```diff
@@ -32,7 +32,7 @@ fn main() -> Result<(), Error> {
     let mut ntrimmed = 0;
     let mut total_reads = 0;
 
-    let reader = Reader::from_file(&input.input_fasta)?;
+    let reader = Reader::from_file(&input.input_fasta).unwrap();
```

Comment:
what was this for?

Reply:
Allowing to to compile?  Because the return value is a result from `anyhow::Error`, not std.  I don't care enough right now understand the details but this way it matches the other code that calls `from_file`, and doesn't fail `cargo check --all-targets`